lnrpc/devrpc/dev.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+import "lightning.proto";
+
+package devrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/devrpc";
+
+service Dev {
+    /* lncli: `importgraph`
+    ImportGraph imports a ChannelGraph into the graph database. Shall not be
+    used for normal operation.
```

Comment:
What does "shall not be used for normal operation" mean? That we don't use the graph data if we import it? Or that its only intended for testing?

Reply:
Ok, so only for development purposes? I think "should only be used for development" is a bit clearer ?

Reply:
Thanks, much better the way you phrased.